bindings/yarp.i

```diff
@@ -42,10 +42,10 @@
   %include "std_vector.i"
 #endif
 
-// Try to make yarp::os::ConstString act like std::string
+// Try to make std::string act like std::string
```

Comment:
This is weird now. :D

Reply:
:rofl:
I'm not sure if this part should be removed from the bindings...